src/seedu/addressbook/data/person/Email.java

```diff
@@ -38,24 +35,9 @@ public static boolean isValidEmail(String test) {
     }
 
     @Override
-    public String toString() {
-        return value;
-    }
-
-    @Override
     public boolean equals(Object other) {
```

Comment:
good job keeping the equals method in individual subclasses to enforce class type equality.